ocs_ci/ocs/ocp.py

```diff
@@ -241,6 +241,24 @@ def apply(self, yaml_file):
         command = f"apply -f {yaml_file}"
         return self.exec_oc_cmd(command)
 
+    def patch(self, resource_name, params, type='json'):
```

Comment:
This function is already part of pr #803 but this code has some enhancement, will resolve the conflict if pr #803 is merged.